src/components/WeeklySummariesReport/FormattedReport.jsx

```diff
@@ -228,6 +228,9 @@ const FormattedReport = ({ summaries, weekIndex, bioCanEdit }) => {
               <span onClick={() => handleGoogleDocClick(googleDocLink)}>
                 <img className="google-doc-icon" src={google_doc_icon} alt="google_doc" />
               </span>
+              <span>
+                <b>&nbsp;&nbsp;{summary.role !== 'Volunteer' && `(${summary.role})`}</b>
```

Comment:
Maybe use CSS will better than using &nbsp;&nbsp. For example, classname the code google, and write the css below:
`.google {
  margin-left: 10px;
}`